mmv1/third_party/terraform/tests/resource_composer_environment_test.go.erb

```diff
@@ -1314,80 +1438,81 @@ resource "google_compute_subnetwork" "test" {
 func testAccComposerEnvironment_composerV2PrivateServiceConnect(envName, network, subnetwork string) string {
 	return fmt.Sprintf(`
 resource "google_composer_environment" "test" {
-	name   = "%s"
-	region = "us-central1"
-
-		config {
-			node_config {
-      	network    			= google_compute_network.test.self_link
-      	subnetwork 			= google_compute_subnetwork.test.self_link
-    	}
-
-  		software_config {
-  		  image_version = "composer-2-airflow-2"
-  		}
-  		private_environment_config {
-				cloud_composer_connection_subnetwork		= google_compute_subnetwork.test.self_link
-  		}
-  	}
+  name   = "%s"
+  region = "us-central1"
+
+    config {
+      node_config {
+        network          = google_compute_network.test.self_link
+        subnetwork       = google_compute_subnetwork.test.self_link
+      }
+
+      software_config {
+        image_version = "composer-2-airflow-2"
+      }
+      private_environment_config {
+        cloud_composer_connection_subnetwork    = google_compute_subnetwork.test.self_link
+      }
+    }
 
 }
 
 resource "google_compute_network" "test" {
-	name                    = "%s"
-	auto_create_subnetworks = false
+  name                    = "%s"
+  auto_create_subnetworks = false
 }
 
 resource "google_compute_subnetwork" "test" {
-	name          = "%s"
-	ip_cidr_range = "10.2.0.0/16"
-	region        = "us-central1"
- 	network       = google_compute_network.test.self_link
-	private_ip_google_access = true
+  name          = "%s"
+  ip_cidr_range = "10.2.0.0/16"
+  region        = "us-central1"
+   network       = google_compute_network.test.self_link
+  private_ip_google_access = true
 }
 
 `, envName, network, subnetwork)
 }
 
+<% unless version == "ga" -%>
```

Comment:
Is this line added intentionally? Any features configured below are beta-only?

Looks like it is failed to compile due to this unclosed block. Need `<% end -%>` to close this block.